flake.lock

```diff
@@ -155,6 +158,21 @@
         "rev": "1c8745c54a239d20b6fb0f79a8bd2628d779b27e",
         "type": "github"
       }
+    },
+    "systems": {
+      "locked": {
+        "lastModified": 1681028828,
+        "narHash": "sha256-Vy1rq5AaRuLzOxct8nz4T6wlgyUR7zLU309k9mBC768=",
+        "owner": "nix-systems",
```

Comment:
I don't really understand, is this necessary for this update to work?

Reply:
This was added automatically by `nix flake update`.

Reply:
Thanks, makes sense